docker/build_capstone.sh

```diff
@@ -16,7 +16,9 @@ git clone --branch=${version} https://github.com/capstone-engine/capstone
 cd capstone
 mkdir build
 cd build
-cmake .. -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_POSITION_INDEPENDENT_CODE=ON -DBUILD_SHARED_LIBS=OFF
+
+# Capstone renamed their build args, so use both until we get a stable 6.0 release from them
+cmake .. -DCMAKE_INSTALL_PREFIX=/usr -DCAPSTONE_BUILD_SHARED_LIBS=ON -DBUILD_SHARED_LIBS=ON
```

Comment:
I think you don't need `BUILD_SHARED_LIBS=ON` as `CAPSTONE_BUILD_SHARED_LIBS` adds `SHARED` to the library (see capstone's CMakeFile.txt:824:

```suggestion
cmake .. -DCMAKE_INSTALL_PREFIX=/usr -DCAPSTONE_BUILD_SHARED_LIBS=ON
```

Reply:
> I think you don't need `BUILD_SHARED_LIBS=ON` as `CAPSTONE_BUILD_SHARED_LIBS` adds `SHARED` to the library (see capstone's CMakeFile.txt:824:

`CAPSTONE_BUILD_SHARED_LIBS` isn't available in the 6.0.0-Alpha1 tag. Actually, @wxrdnx is there a reason we can't just use whatever the HEAD of 'next' is here? That would simplify things.

Reply:
I made the minimum 6.0.0-Alpha5 and required `capstone::capstone_shared` to make sure we get a PIC target to link against.